repo-info/requirements.txt

```diff
@@ -0,0 +1,18 @@
+certifi==2022.9.24
```

Comment:
How did you get this `requirements.txt`? Is is auto-generated? I'm not sure whether we need to list all these packages. Installing one package will automatically install its dependencies.